recipe-server/normandy/recipes/tests/__init__.py

```diff
@@ -108,6 +106,20 @@ def locales(self, create, extracted, **kwargs):
             for locale in extracted:
                 self.locales.add(locale)
 
+    # This should always be before `enabled`
+    @factory.post_generation
+    def approver(self, create, extracted, **kwargs):
+        if extracted:
+            approval = ApprovalRequestFactory(revision=self.latest_revision)
+            approval.approve(extracted, 'r+')
+
+    # This should always be after `approver` as we require approval to enable a recipe
```

Comment:
Yes they are executed in order, as per:
http://factoryboy.readthedocs.io/en/latest/reference.html#post-generation-hooks